02.general/17.dns-providers/docs.en.md

```diff
@@ -428,3 +433,85 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
 | Protocol       | Address                                            |                |
 |----------------|----------------------------------------------------|----------------|
 | DNSCrypt, IPv4 | Provider: `2.dnscrypt-cert.nawala.id` IP: `180.131.144.144`  | <a href="sdns://AQAAAAAAAAAADzE4MC4xMzEuMTQ0LjE0NCDGC-b_38Dj4-ikI477AO1GXcLPfETOFpE36KZIHdOzLhkyLmRuc2NyeXB0LWNlcnQubmF3YWxhLmlk">Add to AdGuard</a> |
+
+<a name="securedns.eu-dns"></a>
+
+### securedns.eu
+
+[DoT, DoH, DNSCRYPT Provider securedns.eu](https://securedns.eu/) DoT, DoH, DNSCRYPT Provider securedns.eu
```

Comment:
It’s better to do it the following way: [securedns.eu](https://securedns.eu/): DoT, DoH, DNSCRYPT Provider